vault/resource_consul_secret_backend.go

```diff
@@ -157,7 +165,7 @@ func consulSecretBackendCreate(d *schema.ResourceData, meta interface{}) error {
 	log.Printf("[DEBUG] Wrote Consul configuration to %q", configPath)
 	d.Partial(false)
 
-	return nil
+	return consulSecretBackendRead(d, meta)
```

Comment:
@benashz We saw this resource wasn't returning with a Read at the end of the Create when we were on a call together. I opted to change it here, but wanted to point it out in case you think this is too out-of-scope for this bug fix.

Reply:
Pretty odd that this was never done before. Looks good to me. It would be good to validate this change outside of the test framework, i.e. run `terraform` directly.

Reply:
Manually tested out both bootstrap and non-bootstrap backend resources, as well as importing, all seemed good! 🥳